.github/workflows/action-image-scan-deploy-dashing.yaml

```diff
@@ -0,0 +1,31 @@
+on: 
+  push:
```

Comment:
I think the `pull_request` trigger is more adequate. It tests every push on Pull Requests. Check https://github.com/open-br/diffbot2_robot/pull/4

Reply:
This is the deployment, it should not occur on the PR, but on pushes on the default branch